packages/sdk/src/evm/common/any-evm-utils.ts

```diff
@@ -978,6 +1052,21 @@ export async function encodeConstructorParamsForImplementation(
           deploymentPresets["Forwarder"] = deploymentInfo;
         }
 
+        return deploymentInfo.transaction.predictedAddress;
+      } else if (p.name && p.name.includes("entrypoint")) {
```

Comment:
is this for smart accounts?

Reply:
Yes, Entrypoint seems to be the only dependency there, for constructor input.